### DISCUSSION TITLE
FOSS4G Asia 2023 scratchpad - let's share the notes.

### DISCUSSION THREAD

**Discussion by hfu** [UNopenGIS/7#326] [General]
# What is this conversation? 
I just thought it would be great to share notes from [FOSS4G Asia 2023](https://foss4g.asia/2023/). 

## See also: 
- #325
- #318

**Comment by hfu** [comment]
## UN Smart Maps Bazaar node in Seoul
I successfully stared an ipfs node in Seoul from my rental apartment network 😉 

12D3KooWM2Rewgn9z5wrj95mXcYuANcX8StmUyenh2mvdXoa6vpb, running on a Raspberry Pi 400, is now geographically located in Seoul.

**Reply by hfu** [reply]
I stopped it and will bring the Raspberry Pi 400 to Community Sprint.

**Comment by hfu** [comment]
今回の会場はソウル市政府関連の施設であることもあり、WiFiがつながっているわけではなく、このため GitHub が使いにくい状況にあります...。

**Reply by hfu** [reply]
ソウル市公共 WiFi (SEOUL) を使えばかなりつながるということが分かりました。

**Comment by hfu** [comment]
# What do you want to do? - Future ideas
Let's brainstorm future ideas to be tested in the UN Smart Maps Group. 

## See Also:
[<img width="916" alt="スクリーンショット 2023-12-02 7 42 15" src="https://github.com/UNopenGIS/7/assets/18297/1550b0c7-b5df-4f9d-858c-e9f5bfe4b11d">](https://speakerdeck.com/hfu/smart-maps-for-the-united-nations-with-the-united-nations)
click the image above to move to the [speakerdeck share](https://speakerdeck.com/hfu/smart-maps-for-the-united-nations-with-the-united-nations).

**Reply by hfu** [reply]
## @hfu 
- #327 
- #328 
- #329 
- #321

**Comment by hfu** [comment]
# What was interesting? - Learning from FOSS4G Asia 2023

**Reply by hfu** [reply]
## @hfu 
- Not directly related or mentioned in FOSS4G Asia, but I think it is worth exploring the concept of [Digital Public Goods](https://www.un.org/techenvoy/content/digital-public-goods) because I think it is worth consolidating and modernizing the concept of "Open Source [for] Geospatial (OSGeo)", "Free and Open Source Software for Geospatial (FOSS4G)", "Open Data", "Open Geospatial Data", "Open Geospatial Technology", and others. 
- [Customer Experience (CX)](https://www.whitehouse.gov/briefing-room/presidential-actions/2021/12/13/executive-order-on-transforming-federal-customer-experience-and-service-delivery-to-rebuild-trust-in-government/) looks like something SMG can explore. A good example is [Chizuemon](https://office-shirado.com/chizuemon/map/#18.14/34.675281/135.525587/136.4/47) developed by @office-shirado. Prof. Venkatesh also advice us to develop customer or domain oriented services if we are to explore possibilities in South Asia or South East Asia. 
- Is [potluck](https://en.wikipedia.org/wiki/Potluck) an metaphor of SMG? Can we say "free-as-in-potluck" with UN Smart Maps?

**Reply by albertkun** [reply]
Prof. Venkatesh also mentioned that we should try to outreach for projects to empower women and girls. I think that should also extend to us putting smart map skills into their hands too- especially to bridge the digital divide.

**Reply by hfu** [reply]
@albertkun I agree. I found similar project in Syria by UNICEF: [A tiny computer lets Syrian refugees learn](https://sowc2015.unicef.org/stories/a-stripped-down-computer-brings-e-learning-to-out-of-school-refugees/)
